applicationset/controllers/applicationset_controller.go

```diff
@@ -57,6 +57,10 @@ const (
 	//   https://github.com/argoproj-labs/argocd-notifications/blob/33d345fa838829bb50fca5c08523aba380d2c12b/pkg/controller/state.go#L17
 	NotifiedAnnotationKey             = "notified.notifications.argoproj.io"
 	ReconcileRequeueOnValidationError = time.Minute * 3
+
+	// LabelKeyAppSetInstance is the label key to use to uniquely identify the apps of an applicationset
+	// The ArgoCD applicationset name is used as the instance name
+	LabelKeyAppSetInstance = "argocd.argoproj.io/appset-instance"
```

Comment:
I know it's more verbose, but for consistency with the existing `argocd.argoproj.io/application-set-refresh`, could we go with `argocd.argoproj.io/application-set-name`?

Reply:
done @crenshaw-dev